scripts/deploy_kakarot.py

```diff
@@ -32,6 +34,8 @@ async def main():
         f"ℹ️  Connected to CHAIN_ID {CHAIN_ID.value.to_bytes(ceil(log(CHAIN_ID.value, 256)), 'big')} "
         f"with RPC {RPC_CLIENT.url}"
     )
+    if NETWORK == "madara":
```

Comment:
it's because the only pre-deployed account is non standard, this may change in the future (actually I could make a PR to madara), but for now it's the only way I've found